packages/packages/libs/editor-ui/src/components/global-dialog/__tests__/event-bus.test.ts

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { waitFor } from '@testing-library/react';
+
+import { closeDialog, type DialogContent, openDialog, subscribeToDialogState } from '../event-bus';
+
+describe( 'EventBus', () => {
```

Comment:
maybe also call the file `subscribers`
```suggestion
describe( 'subscribers', () => {
```